app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/EmbeddedNavigationActivity.java

```diff
@@ -132,6 +132,7 @@ protected void onDestroy() {
     navigationView.onDestroy();
     if (isFinishing()) {
       saveNightModeToPreferences(AppCompatDelegate.MODE_NIGHT_AUTO);
+      AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_AUTO);
```

Comment:
Does setting this before the `Activity` is destroyed still allow it to be considered on re-creation?

Reply:
Yes it does. Mainly because this is "per app" and in the test app closing any activity redirects you to the `MainActivity` i.e. the night mode is kept if you reopen the `Activity`.